apps/web/components/qc/readonly-form-table.tsx

```
@@ -0,0 +1,147 @@
+      } catch {
+        return String(value);
+      }
+    case "number":
+      return typeof value === "number" ? value : Number(value) || String(value);
+    case "file":
+      if (Array.isArray(value)) return value.join(", ");
+      return String(value);
+    default:
+      if (typeof value === "object") return JSON.stringify(value);
+      // Truncate long textareas
+      const s = String(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)